src/GovUk.Education.ExploreEducationStatistics.Content.ViewModels/PublicationCacheViewModel.cs

```diff
@@ -8,6 +8,8 @@ public record PublicationCacheViewModel
 
     public string Title { get; init; } = string.Empty;
 
+    public string Summary { get; set; } = string.Empty;
```

Comment:
We need a faffy deploy ticket for this change, to populate this new value in the cache.

It will be a similar task to https://dfedigital.atlassian.net/browse/EES-5918 except that the request will need to be `DELETE /api/bau/public-cache/publications/publication-json`